Framework/LegacyNexus/inc/MantidLegacyNexus/NeXusFile_fwd.h

```diff
@@ -13,6 +13,7 @@ typedef const char CONSTCHAR;
 
 typedef void *NXhandle; /* really a pointer to a NexusFile structure */
 
+// Should this be 64 in line with NX_MAXNAMELEN?
 typedef char NXname[128];
```

Comment:
Does this need any looking into before this pr goes in?

Reply:
I don't think so, this has no impact on the behaviour of the code, its just that the buffer allocated for NXname looks to be larger than it strictly needs to be.

I'd change it but I'm worried there was a reason this was done in the first place (very old code) and reducing the buffer might lead to access violations.